### ISSUE TITLE
Setting up an auto-archive strategy/policy for GH projects

### ISSUE THREAD

**Issue by widal001** [HHS/grants-product-and-delivery#280]
### Summary

In order to keep the number of active tickets in a project manageable (for searching, filtering, etc.) we might want to adopt an archive policy. Archiving a project item removes them from the UI, but still preserves the project metadata.

### Acceptance criteria

- [x] We've decided a policy around if/when project items should be archived
- [x] We've either set up automation for archiving or created a separate ticket to automate this archiving

**Comment by sarahknoppA6** [comment]
@widal001 was this at some point completed?

**Comment by widal001** [comment]
@sarahknoppA6 No, this still needs to happen, and should actually happen soon, so thanks for flagging! It's relevant to sprint metrics, so I'll add it to this epic: https://github.com/HHS/simpler-grants-gov/issues/6487

**Comment by widal001** [comment]
I set up [this auto-archive workflow](https://github.com/orgs/HHS/projects/17/workflows/55116273) which archives issues in the P&D board after they've been closed for 32 weeks

It archived ~300 items in the Product and Delivery board (see screenshot). We'll want to monitor sprint metrics to make sure that it doesn't have unintended consequences on reporting.

<img width="1108" height="694" alt="Image" src="https://github.com/user-attachments/assets/784175be-0b40-47fa-9605-69bab71b989b" />

**Comment by widal001** [comment]
We've noticed that the data no longer appears for sprints that ended more than >32 weeks ago, but no other noticeable impact. We're calling this Done.
